src/configuration/app/source-operations.md

```diff
@@ -5,9 +5,10 @@ search:
 
 # Source operations
 
-> **note**
+> **warning**
 >
-> Source Operations are currently in beta.  While the syntax is not expected to change, some behavior might in the future.
+> Source Operations are currently in Beta. While the syntax is not expected to change, some behavior might in the future.
+> For example, external code integrations like GitHub, Gitlab or Bitbucket, is not yet compatible with source operations yet.
```

Comment:
"not yet compatible with yet".  Pick one yet. :smile: 

That said, I don't think they're going to be compatible any time soon.  Not until we have bidirectional commit to a linked repo, which is a blocker for a lot of things and I've seen no movement on it.

Honestly I'd consider this two separate warnings.  One that it's in beta (note), and one that it doesn't work with integrations (warning).